charts/boundary/templates/controller-deployment.yaml

```diff
@@ -49,13 +49,15 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       terminationGracePeriodSeconds: 10
+      {{- if .Values.controller.initContainers.enabled }}
```

Comment:
Shall we add the same to database-init-job as well? Also should we rename it to `.Values.job.enabled`?